src/cli/load.test.js

```diff
@@ -76,7 +75,8 @@ describe("cli/load", () => {
           id: "foo/bar",
           repoIds: [makeRepoId("foo", "bar")],
         }),
-        params: defaultParams(),
+        params: null,
```

Comment:
This changes the test more than is needed, and should probably keep the `defaultParams()`.

Because `defaultParams()` sets a value for all options (unlike `partialParams()`), setting it to `null` now removes the alpha and intervalDecay options.

Reply:
Done.